docs/operator-guides/refreshes.md

```diff
@@ -0,0 +1,128 @@
+---
+products: all
+---
+
+# Refreshes
+
+:::info
+
+This page describes a coming-soon feature. Refreshes will be available as of Airbyte v1.0.0.
+
+:::
+
+## Refreshes Overview
+
+As of Airbyte 1.0, Refresh Syncs have been added to the Airbyte platform. A Refresh Sync erases the current cursor you have set for any incremental streams, effectively ‘rewinding’ the sync back to the beginning. This is helpful in the following scenarios:
+
+- The source you're syncing from does not sync record deletions/removals, and you wish to truly mirror the source stream, which would include removing deleted records
```

Comment:
nit: is it really table mirroring if we have duplicates? (b/c resumable full refresh)

Reply:
yeah... this is hard to talk about :(.  I'll remove the word 'truly' at least ;D